pkg/registry/registry_aliyun.go

```diff
@@ -76,6 +77,13 @@ func (r *Registry) GetAliyunTagsWithDetails(image string) (tags []QuayTag, err e
 		tagRequest.Page = requests.NewInteger(page)
 
 		crResp, err := crClient.GetRepoTags(tagRequest)
+		serverError, isAliyunError := err.(*errors.ServerError)
+		if isAliyunError {
+			if strings.Contains(serverError.Message(), "REPO_NOT_EXIST") {
+				r.logger.Log("level", "warn", "message", "Aliyun repository does not exist. Retagger will try to create it")
```

Comment:
It seems like this just make the error message a little nicer. Missing repo creation hasn't been implemented, has it?

Reply:
This was done here:
https://github.com/giantswarm/retagger/pull/287

So we only need to prevent actually erroring out here.

Reply:
Totally forgot about that. Thanks.